package io.ray.runtime.exception;

public class RayTaskException extends RayException {

  public RayTaskException(String message, Throwable cause) {
    super(message, cause);
  }

}
